packages/teleport/src/services/user/types.ts

```diff
@@ -39,4 +39,22 @@ export interface Acl {
   roles: Access;
   sessions: Access;
   events: Access;
+  users: Access;
+}
+
+export interface NewUser {
+  name: string;
+  roles: string[];
+}
+
+export interface StoredUser {
+  name: string;
+  roles: string[];
+  created: Date;
+}
```

Comment:
why types (NewUser/StoredUser)?

`User` would do.

Reply:
revised (renamed existing User -> UserContext) and combined NewUser/StoredUser -> User